### ISSUE TITLE
feat: add `--verbose` flag

### ISSUE THREAD

**Issue by wolfenrain** [VeryGoodOpenSource/very_good_dart_cli#18]
As a developer it would be nice to have a verbose flag so I’m able to better understand what the cli is doing when executing various commands.

**Comment by BeatriceMitchell** [comment]
Unblocked, as logger has log levels now.
